content/posts/ai-crawlers-abuse.md

```diff
@@ -0,0 +1,177 @@
+title: AI Crawlers need to be more respectful
+date: 2024-07-25
+category: Engineering
+tags: ai, crawlers, abuse
+authors: Eric Holscher
+status: published
+image:
+image_credit:
+
+Many AI companies are starting to crawl many pages on the web,
+trying to extract as much information as fast as possible.
+It feels like a new gold rush,
+and in their haste,
+some of these crawlers are being so badly they are causing harm to the sites they are crawling.
+
+At Read the Docs,
+we host documentation for many projects and are generally bot friendly,
+but the behavior of AI crawlers is currently causing us problems.
+We host HTML zip files (``htmlzip``) of documentation for many projects,
+and these can be somewhat large.
+The AI Crawlers are downloading these files repeatedly,
+and without any rate or bandwidth limiting.
```

Comment:
What is actually happening is explained well below. Introducing the problem at a high level would flow nicely here. Again, just an idea of what might work:

```suggestion
We have noticed AI crawlers aggressively pulling content, seemingly without basic
checks against abuse. Bots repeatedly download large files hundreds of
times in a single day, with traffic sent from distributed sources without
rate limiting or bandwidth limitations.
```